socialhome/streams/app/components/LoadingElement.vue

```diff
@@ -0,0 +1,215 @@
+<template>
+    <div class="cssload-loader-inner">
+        <div class="cssload-cssload-loader-line-wrap-wrap">
```

Comment:
Wow, seems pretty complex for a spinner :P What does it look like, can have screenshot / gif? Did you write all the code yourself? If it came from SO or some other resource, please add a comment.

Reply:
Yes, it's quite complex. All aknowledge due, I'm not good enough in CSS to produce that a fancy spinner. I found it on a website. You're right, I should link to the source.